<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1100</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>3</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCheckBox" name="checkBox_Itailic">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>122</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>倾斜(&amp;I)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Bold">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>122</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>加粗(&amp;B)</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_About">
    <property name="geometry">
     <rect>
      <x>662</x>
      <y>0</y>
      <width>711</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>WhatsThisCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{border-radius:15px}</string>
    </property>
    <property name="locale">
     <locale language="Chuvash" country="Russia"/>
    </property>
    <property name="text">
     <string>···</string>
    </property>
    <property name="autoRepeatInterval">
     <number>99</number>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_Input">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>683</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>输入内容(&amp;A)</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Chinese">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>170</y>
      <width>122</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>中文(&amp;C)</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Underline">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>122</width>
      <height>27</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>下划线(&amp;U)</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBox_FontSize">
    <property name="geometry">
     <rect>
      <x>243</x>
      <y>294</y>
      <width>126</width>
      <height>29</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>571</width>
      <height>16777215</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="value">
     <number>35</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_FontSize">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>270</y>
      <width>126</width>
      <height>78</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>字号(&amp;F)</string>
    </property>
    <property name="buddy">
     <cstring>spinBox_FontSize</cstring>
    </property>
   </widget>
   <widget class="QTextBrowser" name="textBrowser_Output">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>470</y>
      <width>591</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="plainTextEdit_Input">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>370</y>
      <width>401</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
